testsuite/mixed-domain/src/test/java/org/jboss/as/test/integration/domain/mixed/eap640/DomainAdjuster640.java

```diff
@@ -287,6 +287,10 @@ public void adjustInfinispanStatisticsEnabled(final List<ModelNode> list, final
 
     private Collection<? extends ModelNode> replaceUndertowWithWeb(final PathAddress subsystem) {
         final List<ModelNode> list = new ArrayList<>();
+
+        // mod_cluster depends on the listener capability so we need to remove the subsystem before we can remove undertow
+        list.add(createRemoveOperation(subsystem.getParent().append(SUBSYSTEM, "modcluster")));
```

Comment:
TBH I think all groups of operations we create in our tests should be run as composite ops as with use of capabilities, this will no longer be possible anyway. Unfortunately, when I convert to composite op, the tests fail with https://gist.github.com/rhusar/c81c6821ba35feb4be5082a6ab7268e2, yet to investigate why.